main/reference/zoe-api/zoe-data-types.md

```diff
@@ -57,21 +63,21 @@ const myAmountKeywordRecord =
 
 ## Invitation
 
-TBD
-
-For most purposes, a good enough approximation is that an Invitation is indeed a kind of Payment.
-
-Technically, an Invitation is a non-fungible eright that can be held in payments or purses, just like any other eright. An Invitation payment would be a Payment holding an Invitation. An Invitation payment is a kind of Payment.
-
-But we almost always just use a Payment holding a single Invitation in order to pass around Invitations, so we normally elide the difference.
+An **Invitation** is a kind of **[Payment](/reference/ertp-api/payment.md)**. It's a non-fungible eright
+that can be held in **Payments** or **[Purses](/reference/ertp-api/purse.md)**, just like any other
+eright. An **Invitation** **Payment** would be a **Payment** holding an **Invitation**. Because we
+almost always just use a **Payment** holding a single **Invitation** in order to pass around
+**Invitations**, the difference is normally elided.
 
 ## InvitationIssuer
 
 **InvitationIssuers** are special types of **[Issuers](/reference/ertp-api/issuer.md)**. Every Zoe
 instance has a single **InvitationIssuer** for the entirety of its lifetime. All **Invitations** come
 from the **[Mint](/reference/ertp-api/mint.md)** associated with the Zoe instance's **InvitationIssuer**.
 
-**InvitationIssuers** have all the methods of regular **Issuers**, but the two methods that qre most often used are **[anIssuer.claim()](/reference/ertp-api/issuer.md#anissuer-claim-payment-optamount)** and **[anIssuer.getAmountOf()](/reference/ertp-api/issuer.md#anissuer-getamountof-payment)**.
+**InvitationIssuers** have all the methods of regular **Issuers**, but the two methods that qre most
```

Comment:
```suggestion
**InvitationIssuers** have all the methods of regular **Issuers**, but the two methods that are most
```

Do you have a spell checker that works with html-formatted text? I wonder what other typos we aren't seeing.

Reply:
I do not, although there's apparently a built-in spell-checker within the doc build architecture.... which is currently broken.

I did a quick search online for a possible solution that didn't have unacceptable overhead, and I didn't see any good candidates. But I agree, this is a concern. I need to figure something out... maybe fix the built-in spell-checker or something.

(And this particular typo has been fixed.)